src/dnx.host/RuntimeEnvironment.cs

```diff
@@ -23,6 +23,12 @@ public RuntimeEnvironment()
             RuntimeType = Type.GetType("Mono.Runtime") == null ? "CLR" : "Mono";
             RuntimeArchitecture = Environment.Is64BitProcess ? "x64" : "x86";
 #endif
+
+            // This is a temporary workaround until we pass a struct with OS information from native code
+            if (Environment.GetEnvironmentVariable(EnvironmentNames.OperatingSystemName) == "1")
```

Comment:
This is confusing. "if the operating system **name** is '1' then ..." . One as in "one core?"